fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -3870,6 +3870,77 @@ struct ShardedRocksDBKeyValueStore : IKeyValueStore {
 	Future<Void> refreshRocksDBBackgroundWorkHolder;
 	Future<Void> cleanUpJob;
 	Future<Void> counterLogger;
+
+	Future<Void> ingestSSTFiles(std::string bulkLoadLocalDir,
```

Comment:
May consider to include the shardedrocksdb support in a separate PR.

Reply:
I'm excluding sharded rocksdb from this PR because of your other suggestion later.

Reply:
Thank you!